.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -75,5 +79,5 @@ For example, `action required: change the API interface of the rule engine`.
 -->
 
 ```release-note
-
+NONE
```

Comment:
I'd not set `NONE` by default, otherwise, folks will just forget to add a release note.

Reply:
fixed :)